### ISSUE TITLE
Monitor Redis Cluster with prometheus data-source.

### ISSUE THREAD

**Issue by vineelyalamarthy** [RedisGrafana/grafana-redis-datasource#275]
We want to monitor Redis-Cluster , I really like the visual representation in this chart. But want to use the prometheus as a data source. Does this dashboard allow Grafana as a data source.

**Comment by mikhail-vl** [comment]
@vineelyalamarthy You can choose any data source for any panel in the default dashboard and update per your needs.

**Comment by vineelyalamarthy** [comment]
Sorry, didn't get you, we can use prometheus as a data source or not? Please confirm @mikhail-vl

**Comment by mikhail-vl** [comment]
@vineelyalamarthy Yes, you can. Just copy the dashboard and use Prometheus data source.
